examples/src/main/proto/error_details.proto

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
Copied from the original file, not sure about your copyright section policy, I'll update it.